src/meta/proto-conv/tests/it/user_proto_conv.rs

```diff
@@ -228,6 +290,59 @@ fn test_load_old_user() -> anyhow::Result<()> {
 
     // UserStage is loadable
     {
+        let user_stage_info_v4: Vec<u8> = vec![
+            10, 24, 115, 51, 58, 47, 47, 109, 121, 98, 117, 99, 107, 101, 116, 47, 100, 97, 116,
+            97, 47, 102, 105, 108, 101, 115, 16, 1, 26, 100, 10, 98, 10, 96, 18, 24, 104, 116, 116,
+            112, 115, 58, 47, 47, 115, 51, 46, 97, 109, 97, 122, 111, 110, 97, 119, 115, 46, 99,
+            111, 109, 26, 9, 109, 121, 95, 107, 101, 121, 95, 105, 100, 34, 13, 109, 121, 95, 115,
+            101, 99, 114, 101, 116, 95, 107, 101, 121, 42, 8, 109, 121, 98, 117, 99, 107, 101, 116,
+            50, 11, 47, 100, 97, 116, 97, 47, 102, 105, 108, 101, 115, 58, 13, 109, 121, 95, 109,
+            97, 115, 116, 101, 114, 95, 107, 101, 121, 160, 6, 4, 168, 6, 1, 34, 20, 8, 1, 16, 128,
+            8, 26, 1, 124, 34, 2, 47, 47, 40, 2, 160, 6, 4, 168, 6, 1, 42, 8, 10, 3, 32, 154, 5,
+            16, 142, 8, 50, 4, 116, 101, 115, 116, 160, 6, 4, 168, 6, 1,
+        ];
+
+        let p: pb::UserStageInfo =
+            common_protos::prost::Message::decode(user_stage_info_v4.as_slice())
+                .map_err(print_err)?;
+
+        let got = mt::UserStageInfo::from_pb(p).map_err(print_err)?;
+
+        println!("got stage: {:?}", got);
+
+        let want = test_user_stage_info();
+
+        assert_eq!(want, got);
+    }
+
+    {
+        // Stage on Google Cloud Storage
+        let user_stage_info_gcs = vec![
+            10, 26, 103, 99, 115, 58, 47, 47, 109, 121, 95, 98, 117, 99, 107, 101, 116, 47, 100,
+            97, 116, 97, 47, 102, 105, 108, 101, 115, 16, 1, 26, 81, 10, 79, 26, 77, 10, 30, 104,
+            116, 116, 112, 115, 58, 47, 47, 115, 116, 111, 114, 97, 103, 101, 46, 103, 111, 111,
+            103, 108, 101, 97, 112, 105, 115, 46, 99, 111, 109, 18, 9, 109, 121, 95, 98, 117, 99,
+            107, 101, 116, 26, 11, 47, 100, 97, 116, 97, 47, 102, 105, 108, 101, 115, 34, 13, 109,
+            121, 95, 99, 114, 101, 100, 101, 110, 116, 105, 97, 108, 160, 6, 4, 168, 6, 1, 34, 20,
+            8, 1, 16, 128, 8, 26, 1, 124, 34, 2, 47, 47, 40, 2, 160, 6, 4, 168, 6, 1, 42, 8, 10, 3,
+            32, 154, 5, 16, 142, 8, 50, 4, 116, 101, 115, 116, 160, 6, 4, 168, 6, 1,
+        ];
```

Comment:
As I said in the previous comment, mark the message with a version. Otherwise, you will never know when to remove this test case.

Reply:
I'll change it to 
```suggestion
        let user_stage_info_v4_gcs = vec![
            10, 26, 103, 99, 115, 58, 47, 47, 109, 121, 95, 98, 117, 99, 107, 101, 116, 47, 100,
            97, 116, 97, 47, 102, 105, 108, 101, 115, 16, 1, 26, 81, 10, 79, 26, 77, 10, 30, 104,
            116, 116, 112, 115, 58, 47, 47, 115, 116, 111, 114, 97, 103, 101, 46, 103, 111, 111,
            103, 108, 101, 97, 112, 105, 115, 46, 99, 111, 109, 18, 9, 109, 121, 95, 98, 117, 99,
            107, 101, 116, 26, 11, 47, 100, 97, 116, 97, 47, 102, 105, 108, 101, 115, 34, 13, 109,
            121, 95, 99, 114, 101, 100, 101, 110, 116, 105, 97, 108, 160, 6, 4, 168, 6, 1, 34, 20,
            8, 1, 16, 128, 8, 26, 1, 124, 34, 2, 47, 47, 40, 2, 160, 6, 4, 168, 6, 1, 42, 8, 10, 3,
            32, 154, 5, 16, 142, 8, 50, 4, 116, 101, 115, 116, 160, 6, 4, 168, 6, 1,
        ];
```
This should illustrate the version.

Reply:
I think renaming `test_user_stage_info` to `s3_stage_info`, and `test_user_stage_info_gcs` to `gcs_stage_info` is better.